client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -299,25 +310,36 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 					closeModal={ closeUpgradeModal }
 					checkout={ goToCheckout }
 				/>
-				<WebPreview
-					showPreview
-					showClose={ false }
-					showEdit={ false }
-					externalUrl={ siteSlug }
-					showExternal={ true }
-					previewUrl={ previewUrl }
-					loadingMessage={ translate(
-						'{{strong}}One moment, please…{{/strong}} loading your site.',
-						{
-							components: { strong: <strong /> },
-						}
-					) }
-					toolbarComponent={ PreviewToolbar }
-					siteId={ site?.ID }
-					url={ site?.URL }
-					translate={ translate }
-					recordTracksEvent={ recordTracksEvent }
-				/>
+				{ isEnabledStyleSelection ? (
```

Comment:
By the way, I think we want to show the old theme preview component if the theme has no style variations? Or perhaps this logic can go in a followup PR (if we want to do it).

Reply:
Let's follow up on this. I think we will need a clearer course of action here because I remember also being discussion as to having some sort of placeholder text (even for generated designs) cc: @SaxonF